portal-web/test/functional/com/liferay/portalweb/paths/portlet/publications/PublicationsChanges.path

```diff
@@ -156,6 +156,34 @@
 	</td>
 <td></td>
 </tr>
+
+<!--SHARE ACCESS-->
+
+<tr>
+	<td>COPY_URL</td>
+	<td>//div[contains (@class,'input-group-item-shrink')]//button[contains(text(),'Copy')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ENABLE_SHARE_LINK_TOGGLE_SWITCH</td>
+	<td>//span[contains(.,'${key_switchText}')]/..//span[contains(@class, 'toggle-switch-check-bar')]//input[contains(@type, 'checkbox')]</td>
```

Comment:
Can you update the switch text with real text instead of the variable? Since it's fixed, we should make it easy to maintain. After that, we won't need to type it every time we use it. if it changes, updating the locator is enough.